packages/auth/src/adapter.ts

```diff
@@ -1,42 +1,207 @@
-import { prisma, upsertDiscordAccount } from '@answeroverflow/db';
-import { PrismaAdapter } from '@next-auth/prisma-adapter';
+import { accounts, db, users, verificationTokens } from '@answeroverflow/db';
 import { getDiscordUser } from '@answeroverflow/cache';
-import type { Adapter, AdapterAccount } from 'next-auth/adapters';
+import { sessions, tenantSessions } from '@answeroverflow/db/src/schema';
+import { upsertDiscordAccount } from '@answeroverflow/db/src/discord-account';
+import { and, eq } from 'drizzle-orm';
+import type { Adapter } from '@auth/core/adapters';
+import { randomUUID } from 'node:crypto';
 
 export const extendedAdapter: Adapter = {
```

Comment:
Add comment about us needing to make our own adapter due to drizzle not matching prisma adapter